python/lib/dcoscli/dcoscli/package/main.py

```diff
@@ -73,7 +73,7 @@ def _cmds():
         cmds.Command(
             hierarchy=['package', 'install'],
             arg_keys=['<package-name>', '--package-version', '--options',
-                      '--cli', '--global', '--app', '--yes'],
+                      '--cli', '--global', '--app', '--yes', '--manager-id'],
```

Comment:
I think we agreed to hide the `managerId` field from the user help menu. Is that what's being done here?

Reply:
I don't think it's easily doable, the library it's using (docopt) relies on the help menu to parse available commands and flags.

Though I'm surprised that this change works if you didn't change https://github.com/dcos/dcos-core-cli/blob/master/python/lib/dcoscli/dcoscli/data/help/package.txt.